README.md

```diff
@@ -1,6 +1,6 @@
 # js-eval
 ```sh
-$ echo '{ "environment": "node-cjs", code: "1 + 1" }' | docker run -i js-eval
```

Comment:
ok done, I can reput the echo version there if you really prefer, there was missing quotes around `code`